addons/headless/XEH_postInit.sqf

```diff
@@ -11,6 +11,6 @@ if (hasInterface && !isServer) exitWith {};
         };
     } else {
         // Register HC (this part happens on HC only)
-        ["ACE_HeadlessClientJoined", [player]] call EFUNC(common,globalEvent);
+        ["ACE_HeadlessClientJoined", [player]] call (ace_common_globalEvent);
```

Comment:
It should be `ace_common_fnc_globalEvent`.
And I guess you can ommit the parantheses.